python/instrumentation/openinference-instrumentation-anthropic/src/openinference/instrumentation/anthropic/_wrappers.py

```diff
@@ -326,11 +328,16 @@ def _get_llm_token_counts(usage: "Usage") -> Iterator[Tuple[str, Any]]:
     yield LLM_TOKEN_COUNT_COMPLETION, usage.output_tokens
 
 
-def _get_llm_model(arguments: Mapping[str, Any]) -> Iterator[Tuple[str, Any]]:
+def _get_llm_model_arg(arguments: Mapping[str, Any]) -> Iterator[Tuple[str, Any]]:
     if model_name := arguments.get("model"):
         yield LLM_MODEL_NAME, model_name
 
 
+def _get_llm_model(message: "Message") -> Iterator[Tuple[str, Any]]:
```

Comment:
little confused by the stuff above, when does one get used and when does the other get used?

Reply:
i have renamed them for better clarity